app/views/candidate_interface/course_choices/go_to_find.html.erb

```diff
@@ -0,0 +1,17 @@
+<% content_for :title, t('page_titles.find_a_course') %>
+<% content_for :before_content, govuk_back_link_to(candidate_interface_choose_path) %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <h1 class="govuk-heading-xl">
+      <%= t('page_titles.find_a_course') %>
+    </h1>
+
+    <p class="govuk-body-l">Search for courses near you on Find postgraduate teacher training.</p>
+
+    <%= govuk_button_link_to 'Start now', 'https://find-postgraduate-teacher-training.education.gov.uk' %>
```

Comment:
Looks like from the design (https://bat-design-history.netlify.com/apply-for-teacher-training/pick-a-course#find-a-course-when-course-isnt-known) this should be a GOV.UK Design System start button:

See https://design-system.service.gov.uk/components/button/#start-buttons

<img width="841" alt="Screenshot 2020-02-25 at 08 59 45" src="https://user-images.githubusercontent.com/42817036/75231355-2e8a5d80-57ad-11ea-87e0-722363751382.png">

Reply:
Done. Thanks!